### ISSUE TITLE
Adjustable frequency of Saving Trains

### ISSUE THREAD

**Issue by OmegaPaladin** [Railcraft/Railcraft#1748]
**Is your feature request related to a problem? Please describe.**
My server is suffering from frequent block lag and I am seeing messages about skipping ticks.  I also am seeing the saving trains message at least once per minute, and I suspect this is related.  This is linked to #1668 but not the phantom train issue.  

**Describe the solution you'd like**
A config setting for the frequency of saving trains.  I'd like to get down to every 10-15 minutes, like the save-all script I have running.

**Describe alternatives you've considered**
A flat decrease in train saving frequency would also work, but I'm not sure if that would cause it's own issues for some users.

**Comment by liach** [comment]
afaik they are not related. you need a profiler result like a spark one to confirm that the issue is actually caused by trains saving.

**Comment by liach** [comment]
moreover, train saving is bound to server autosave, which you can already configure. a custom lower saving rate from railcraft will lead to errors

**Comment by OmegaPaladin** [comment]
The reason I suspected that the Saving trains was the issue is that I would have a solid 20 TPS from tickprofiler, but still get lag and messages about skipping ticks.  Now, as far as the server autosave, I have the server running save-all every 30 minutes, but as I said the saving trains message appears every minute.  I searched the minecraft wiki and elsewhere for more information, but I could not find another reference to autosave frequency.
